test/header_parser_tests/typedef.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+typedef void (*NamedFunctionProto)();
+
+struct Struct1
```

Comment:
Please also add the pattern from the issue https://github.com/dart-lang/ffigen/issues/73.

```c
typedef struct _Foo {
  // ...
} Foo;
```

Reply:
The tests in `typedef.h` are only for function pointers currently, Just to check that we reuse a typedef instead of recreating it.

Reply:
#73 is related to which name we should use (from the struct namespace or `global namespace`), I will add tests for in its own PR.

Reply:
👍